microsoftgraph/microsoft-graph-explorer-v2
# Feature: change default link to Devx Api

## Overview
Allows the changing of the GE API base url on the fly to help in testing the versions being published.
Fixes #330 

### Demo


![image](https://user-images.githubusercontent.com/58787602/72893633-91aa4f80-3d2a-11ea-8c39-1b461f15f5e8.png)



### Notes

Leverages the query parameters in the url. Deleting the added parameters defaults it to the original url

## Testing Instructions

* Run the app
* Add a query parameter at the end pointing to the desired endpoint e.g /?devx-api=https://graphexplorerapi.azurewebsites.net

* In the network tab, notice that it fetches the new url.
